internal/providers/terraform/aws/cloudfront_distribution.go

```diff
@@ -0,0 +1,67 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetCloudfrontDistributionRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_cloudfront_distribution",
+		RFunc: NewCloudfrontDistribution,
+	}
+}
+
+func NewCloudfrontDistribution(d *schema.ResourceData, u *schema.ResourceData) *schema.Resource {
+	return &schema.Resource{
+		Name: d.Address,
+		SubResources: []*schema.Resource{
+			regionalDataOutToOrigin(u),
+		},
+	}
+}
+
+func regionalDataOutToOrigin(u *schema.ResourceData) *schema.Resource {
+	resource := &schema.Resource{
+		Name:           "Regional Data Transfer Out to Origin",
+		CostComponents: []*schema.CostComponent{},
+	}
+
+	// regionMap structure is: aws grouped name -> [pricing region , usage data key]
+	regionsMap := map[string][]string{
+		"United States, Mexico, & Canada":    []string{"United States", "united_states_data_transfer_origin_gb"},
+		"Europe & Israel":                    []string{"Europe", "europe_data_transfer_origin_gb"},
+		"South Africa, Kenya, & Middle East": []string{"South Africa", "south_africa_data_transfer_origin_gb"},
+		"South America":                      []string{"South America", "south_america_data_transfer_origin_gb"},
+		"Japan":                              []string{"Japan", "japan_data_transfer_origin_gb"},
+		"Australia & New Zealand":            []string{"Australia", "australia_data_transfer_origin_gb"},
+		"Hong Kong, Philippines, Singapore, South Korea, Taiwan, & Thailand": []string{"Asia Pacific", "asia_pacific_data_transfer_origin_gb"},
```

Comment:
Do you think we should truncate this? Either

Hong Kong, Philippines, Singapore…
or
Hong Kong, Philippines, other Asia Pacific

Reply:
I didn't get what you mean. It's what is shown on AWS pricing page. You mean we change it to sth shorter like `Hong Kong, Philippines, other Asia Pacific`?